pages/2_Data.py

```diff
@@ -20,22 +20,42 @@
 def change_photo_state():
     st.session_state["photo"]="done"
 
-uploaded_file = col2.file_uploader("Choose a CSV file", on_change=change_photo_state)
-file_images = []
-url_column_name = "change to the name of the column with url listing"
+uploaded_file = col2.file_uploader("Choose a CSV file", on_change=None)
```

Comment:
We won't deal with csv file directly but it's used only for storing data internally. User should  register one dataset at a time.

Reply:
I had already made some changes but they came only visible hear after syncing my fork. So those lines were already deleted. But as we do not yet have such such a csv file, I simply made up one as you suggested earlier. Now, you you want to make changes to this list:

![Screenshot from 2023-02-24 17-00-57](https://user-images.githubusercontent.com/93677680/221214470-dc3c25cc-af86-4d93-b7ef-cbb5f5798b07.png)

What changes do you wish to make?

Reply:
You could dummy from:

https://github.com/fastai/fastai/blob/master/nbs/04_data.external.ipynb